src/ScoutingMain.elm

```diff
@@ -67,20 +72,30 @@ stylishPage position title teamNumber page =
             , Font.color Colors.veryLightBlue
             , Font.size size
             ]
+
+        findColor : String -> Element.Color
+        findColor alliance =
+            if String.contains "Blue" alliance then
+                Colors.blue
+
+            else if String.contains "Red" alliance then
+                Colors.red
+
+            else
+                Colors.blue
```

Comment:
`findColor` is defined twice for no reason (it doesn't use any local closure).
If you want it in multiple scopes just define it in the global scope.